app/repositories/migrations/schema6.ts

```diff
@@ -0,0 +1,224 @@
+// schema version
+const schemaVersion = 6;
```

Comment:
`schema6` was already used by https://github.com/Plant-for-the-Planet-org/treemapper/pull/302. Please change this to `schema7` instead and fix the merge conflicts with the current develop branch.

Reply:
yes